src/canvas.py

```diff
@@ -20,8 +22,15 @@ def __init__(self, parent):
         try:
             style_path = available_styles[stylename]
```

Comment:
I actually had it defined like this earlier, but the line became too long with the variable name I was using. Instead of splitting the line, I thought it works just as well by defining the stylename beforehand, especially as that's kinda readable. I'm personally always inclined to try and be a bit verbose.

This is however not that unreadable either, and I just used stylename here as well, there's no reason to use a seperate variable for the default style. Now it just says that stylename is whatever the user has in their config, and if that doesn't work, stylename is the default style from the template. That works for me.